examples/ibc.yaml

```diff
@@ -0,0 +1,46 @@
+ibc-0:
+  validators:
+    - coins: 10cro
+      staked: 10cro
+      base_port: 26650
+    - coins: 10cro
+      staked: 10cro
+  accounts:
+    - name: relayer
+      coins: 100cro
+ibc-1:
+  validators:
+    - coins: 10cro
+      staked: 10cro
+      base_port: 26750
+    - coins: 10cro
+      staked: 10cro
+      base_port: 26760
+  accounts:
+    - name: relayer
+      coins: 100cro
+relayer:
+  global:
+    timeout: 10s
+    light-cache-size: 20
+  # chains field will be populated automatically
+  paths:
+    demo:
+      src:
+        chain-id: ibc-0
+        client-id: demo0client
+        connection-id: demo0connection
+        channel-id: demo0channel
+        port-id: transfer
+        order: unordered
```

Comment:
why unordered?

Reply:
no particular reason for that. I guess unordered have better performance?

Reply:
https://github.com/cosmos/ics/blob/master/spec/ics-020-fungible-token-transfer/README.md -- seems it's required for ICS20 fungible transfers?
> Both machines A and B accept new channels from any module on another machine, if and only if:

> The channel being created is unordered.
> The version string is ics20-1.